chatkit-core/src/test/kotlin/com/pusher/chatkit/SynchronousChatManagerFunctionalTestUtil.kt

```diff
@@ -8,17 +8,6 @@ import com.pusher.util.asSuccess
 import elements.Error
 import java.util.concurrent.Future
 
-internal fun chatForFunctionalTest(
-    userId: String,
-    platformClientFactory: PlatformClientFactory = DefaultPlatformClientFactory()
-) =
-        SynchronousChatManager(
```

Comment:
let's rename the file to `FunctionalTestUtil.kt` as now it has nothing to do with `SynchronousChatManager` which is gone

Reply:
👍

Reply:
👍